lib/components/steps/Home.tsx

```diff
@@ -0,0 +1,107 @@
+import {Box, Button, Stack, Typography} from '@mui/material';
+import TABLE from '../../components/assets/svg/table.svg';
+import X from '../../components/assets/svg/x.svg';
+import {ModalLayout} from "../layout/ModalLayout.tsx";
+import StyledIconButton from '../common/StyledIconButton.tsx';
+import {useCdeContext} from "../../CdeContext.tsx";
+import {STEPS} from "../../models.ts";
+import {processMappingFile, validateMappingFile} from "../../services/csvService.ts";
+
+
+function Home() {
+    const {
+        setStep,
+        labName,
+        handleClose,
+        cdeFileMapping,
+        setLoadingMessage,
+        setErrorMessage,
+        setMapping
+    } = useCdeContext();
+
+
+    const handleStartMapping = async () => {
+        if (!cdeFileMapping) {
+            setErrorMessage('Mapping file not found')
```

Comment:
The context provides a setErrorMessage that will show a snackbar when the message is not null, for 3 seconds. (Not in figma, feel free to remove if not useful)